client/src/components/Spinner/Spinner.component.tsx

```diff
@@ -0,0 +1,38 @@
+import { Center, Spinner as PageSpinner } from '@chakra-ui/react'
```

Comment:
nit: the name `PageSpinner` doesn't make its distinction from `Spinner` clear. I'd suggest `ChakraSpinner` to make it clear that `Spinner` is wrapping a ChakraUI component.